packages/tradershub/src/routes/TradersHubRoute/TradersHubRoute.tsx

```diff
@@ -26,11 +25,11 @@ const TradersHubRoute = () => {
     const isSwitcherVisible = isDIEL && isReal;
     const isTotalAssetsVisible = hasActiveDerivAccount || isDemo;
 
-    if (isMobile)
+    if (!isDesktop)
         return (
-            <div className='pt-16'>
+            <div className='p-20'>
                 <div className='flex items-end justify-between pb-24'>
-                    <div>
+                    <div className='flex flex-col'>
```

Comment:
Any reason to add this here 🤔 . By default HTML already stack the elements on top of each other. Unless somewhere in the wrapper flex is being called/

Reply:
There is another div on top which is row, so i have to specify this